gdkwayland/README.md

```diff
@@ -36,7 +36,7 @@ Avoid mixing versioned and git crates like this:
 # This will not compile
 [dependencies]
 gdk-wayland = "0.15"
```

Comment:
I'd suggest to rename it everywhere to `gdkwayland`. The crate is named without the dash too, so that's how it would end up on crates.io.